### ISSUE TITLE
check imports

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#16]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/16) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

sometimes we have e.g, 

```
#!python

import dateutils
from dateutils import ... as ...
```

We could remove all not needed import statements

**Comment by ReimarBauer** [comment]
**Original comment by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

What is the desired style for imports?
Shall we always use 
  from xyc import a,b,c
or use
  import xyz
  xyz.a()

Is there a PEP about this? Personally I like xyz.a() as one sees where the function is defined. Some people use a IDE for that...?

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

PEP8 style is a different issue
https://www.python.org/dev/peps/pep-0008/#imports

we can do from xyc import a, b, c 

That issue is more about to remove not used imports from the modules or at least verify that both types are needed.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in ReimarBauer/mss (pull request #14)

improved imports based on module names, fixes #16
